content/docs/command-reference/exp/run.md

```diff
@@ -100,7 +101,7 @@ the custom Git reference (in `.git/refs/exps`), similar to a branch.
 
 </details>
 
-## Queueing and parallel execution
+## Queueing and background execution
```

Comment:
WDYT of this conceptual change?

Reply:
I think this should be left alone for the same reasons you noted below. We don't actually run anything in the background, the user can do that themselves if they want using `&` in bash, or by just running multiple terminal instances, but DVC does not behave like any kind of background process/daemon in typical use cases.

Reply:
Rolled back to "parallel" for now.